docs/roman/ramp_fitting/description.rst

```diff
@@ -156,64 +118,78 @@ sufficient; they are given as:
 | 100               |                        | 10       |
 +-------------------+------------------------+----------+
 
-Segment-specific Computations:
-------------------------------
-The variance of the slope of a segment due to read noise is:
+Segment-Specific Computations
+-----------------------------
+
+The segment fitting implementation is based on Section 5 of Casertano et.
+al. 2022. If there is only one segment, no fitting is performed.
```

Comment:
I think of a 'segment' as a block of consecutive resultants with no jumps in between.  One segment is fine; e.g., this is the case for a ramp with no cosmic rays.  A segment with one resultant isn't fine and gets skipped.

Reply:
Nope, I got the terminology wrong. Trying again...